images/nginx/rootfs/build.sh

```diff
@@ -94,6 +94,7 @@ clean-install \
   authbind \
   dumb-init \
   gdb \
+  valgrind \
```

Comment:
👍 I am assuming you will remove it from https://github.com/kubernetes/ingress-nginx/blob/master/rootfs/Dockerfile#L23 when updating base Nginx image for the controller.

Reply:
yes